core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.java

```diff
@@ -87,7 +87,9 @@ private void initTTS(final OnInitSucceedListener onInitSucceedListener) {
         onInitSucceedListener.onInitSucceed();
       } else {
         Log.e(TAG_KIWIX, "Initialization of TextToSpeech Failed!");
-        //TODO: Surface to user
+        Toast.makeText(context,
+          context.getResources().getString(R.string.texttospeech_initialization_failed),
```

Comment:
Toast.makeText has a method that just takes a string resourceId, you don't need to call `getString` and you don't need to call `getResources` even if you did need to call `getString`, context has a `getString` method

Reply:
I'll revert it. I did this because the toasts other than the ones I put here also used the same pattern. I'll change this in those ones too.